src/media-segment-request.js

```diff
@@ -971,7 +971,7 @@ export const mediaSegmentRequest = ({
   }
 
   const segmentRequestOptions = videojs.mergeOptions(xhrOptions, {
-    uri: segment.resolvedUri,
+    uri: segment.part && segment.part.resolvedUri || segment.resolvedUri,
```

Comment:
we don't actually use uri anywhere :shrug: but for completeness sake I changed it.